gson/src/main/java/com/google/gson/internal/Excluder.java

```diff
@@ -198,11 +198,7 @@ private boolean excludeClassChecks(Class<?> clazz) {
           return true;
       }
 
-      if (isAnonymousOrNonStaticLocal(clazz)) {
-          return true;
-      }
-
-      return false;
+    return isAnonymousOrNonStaticLocal(clazz);
```

Comment:
The indentation here is inconsistent with the indentation of the other statements of this method. Though maybe it would be best to just fix the indentation of `excludeClassChecks`, `excludeClass` and `excludeClassInStrategy` to use the expected 2 spaces (instead of 4).